docs/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests.md

```diff
@@ -174,14 +177,17 @@ Up to this point, the code shown is just performing regular Http requests, but t
 
 ## Additional resources
 
-- **Using HttpClientFactory in .NET Core** \
+- **Using HttpClientFactory in .NET Core**  
   [https://docs.microsoft.com/aspnet/core/fundamentals/http-requests](/aspnet/core/fundamentals/http-requests)
 
-- **HttpClientFactory GitHub repo** \
+- **HttpClientFactory GitHub repo**  
```

Comment:
```suggestion
- **HttpClientFactory GitHub repository**  
```

Reply:
Technically the repo is aspnet/Extensions. So maybe say HttpClientFactory source code (GitHub)?